dali/operators/numba_function/numba_func.cc

```diff
@@ -112,7 +112,10 @@ This function must work in Numba ``nopython`` mode.)code", DALI_PYTHON_OBJECT)
 This function is invoked once per batch. Also this function must work in Numba ``nopython`` mode.)code",
                   DALI_PYTHON_OBJECT, nullptr)
   .AddOptionalArg("batch_processing", R"code(Determines whether the function is invoked once per batch or
-separately for every sample in the batch.)code", false);
+separately for every sample in the batch.
+
+The best performance for the CPU is obtained when this option is turned off. When operator 
+is working in per-sample basis multithreading is utilized, which may increase perfomance.)code", false);
```

Comment:
When ``batch_processing`` is set to ``True``, the function processes the whole batch. It is necessary if the function has to perform cross-sample operations and may be beneficial if significant part of the work can be reused. For other use cases, specifying ``False`` and using per-sample processing function allows the operator to process samples in parallel.